.github/workflows/rust.yml

```diff
@@ -6,6 +6,9 @@ on:
   pull_request:
   merge_group:
 
+env:
+  RUSTFLAGS: -Aunexpected_cfgs
```

Comment:
The `unexpected_cfgs` error was added in Rust 1.80, but this release is forking off a past release, v21.2.0, that doesn't compile when that error is enabled, for reasons that are not material.

On the `main` branch the things causing the error to occur will, or have been already, fixed. But for the sake of a patch release that has the goal of making a small change on a past release, silencing the error is more appropriate.